models/dalec/R/write.configs.dalec.R

```diff
@@ -114,6 +137,157 @@ write.config.DALEC <- function(defaults, trait.values, settings, run.id) {
     }
   }
   
+  ### INITIAL CONDITIONS
```

Comment:
There's a good bit here that looks similar to the logic that will be in other pool based models. I wonder if there are parts of this that can be extracted and generalized for reuse in other models